### ISSUE TITLE
2020/0528_tradwiki

### ISSUE THREAD

**Issue by prtksxna** [samwilson/website#3]
Commenting about: https://samwilson.id.au/2020/0528_tradwiki.html

Community spirit and treating everyone equally are core values of anarchism. I don't know enough about _Prussian civil servant_ though. Any suggested reading?

**Comment by samwilson** [comment]
No, I don't know what to read, but would like to find something related. [Here's](https://en.wikipedia.org/wiki/Love_on_a_Branch_Line_(novel)) a light-hearted British take on some of the same themes.

**Comment by prtksxna** [comment]
Thanks for the recommendation 🙏🏽 Not available on Wikisource or Amazon India though 😞
